src/components/orders/DetailsTable/index.tsx

```diff
@@ -333,6 +458,34 @@ export function DetailsTable(props: Props): JSX.Element | null {
               <GasFeeDisplay order={order} />
             </td>
           </tr>
+          <tr>
+            <td>
```

Comment:
nit: I think in this case, it would be justified to create a subcomponent for the appData given only the `appData` is required to show it, and this adds quite some code both markup and some hooks and fetch logic. This could be encapsulated and hidden behind a small component

Reply:
Yes, we should encapsulate the logic. I believe it makes sense to do it when we start working on https://github.com/cowprotocol/explorer/issues/160 as we would reuse the logic here to decode any appdata hash.